modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LinkTag.java

```diff
@@ -162,6 +170,7 @@ protected void cleanUp() {
 		_href = null;
 		_icon = null;
 		_label = null;
+		_localizeTitle = false;
```

Comment:
This would change the default behavior of the link tag, previously we always localized. There are currently 78 usages of `<clay:link>` and 11 usages of `LinkTag` in DXP. This change would almost certainly cause a lot of regressions. We should keep default to `true`. We can also be confident that localization turned on is by far the most common use case.